modules/gapi/src/compiler/gstreaming_priv.hpp

```diff
@@ -56,6 +56,7 @@ class GAPI_EXPORTS GStreamingCompiled::Priv
 
     void setInInfo(const GTypesInfo& info) { m_in_info = std::move(info); }
     const GTypesInfo& inInfo() const { return m_in_info; }
+    bool isDesync() { return m_exec->isDesync(); };
```

Comment:
@dmatveev Is there any graceful way to check whether graph contains `desync` ? 
We didn't find :(

Reply:
Removed. All logic in gstreamingexecutor now.